datasette/static/sort_utils.js

```diff
@@ -0,0 +1,66 @@
+function sort() {
+  var sorting_options = document.getElementsByName("sorting_options")[0];
+  var checked_options = sorting_options.getElementsByTagName("input");
+  let selected_options = [];
+  for (let input of checked_options) {
+    if (input.type == "checkbox" && input.checked == true) {
+      try {
+        var type = document.querySelector(
+          `input[name="${input.name}_t"]:checked`
+        ).value;
+      } catch (e) {
+        alert(`You need to select an option for "${input.name}"!`);
+        return;
+      } finally {
+        let s_option = { name: input.name, value: `${type}` };
+        selected_options.push(s_option);
+      }
+    }
```

Comment:
nit: reverse this condition and add `continue;` to avoid one level of indentation:
```suggestion
    if (input.type != "checkbox" || input.checked != true) {
      continue;
    }
    try {
      var type = document.querySelector(
        `input[name="${input.name}_t"]:checked`
      ).value;
    } catch (e) {
      alert(`You need to select an option for "${input.name}"!`);
      return;
    } finally {
      let s_option = { name: input.name, value: `${type}` };
      selected_options.push(s_option);
    }
```
Also, avoid using `alert()` as this is blocking the whole browser. You might want to append an error message somewhere in the DOM.